README.md

```diff
@@ -72,44 +72,47 @@ The complete flow is as follows:
 Calcite reads the schema information from storage, and the Executor stores a reference to Data Manager for buffer/storage access during a query. 
 
 The return from RelAlgExecutor is a ResultSet object which can be converted to Arrow and to pandas:
-```
+```python
      df = res.to_arrow().to_pandas()
 ```
 
 ## Examples
 
 Standalone examples are available in the `examples` directory. Most examples run via Jupyter notebooks. 
 
+
 ## Build
 
-HDK includes components from external libraries as submodules. To clone the project, either use `git clone --recurse-submodules` to initially clone the repo, or clone without using flags and then run:
+### Dependencies 
 
-```
-git submodule init
-git submodule update
+Miniconda installation required. (Anaconda may produce build issues.) Use one of this [miniconda installers](https://docs.conda.io/en/latest/miniconda.html).
+
+Conda environments are used for HDK development. Use the yaml file in `omniscidb/scripts/`:
+
+```bash
+     conda env create -f omniscidb/scripts/mapd-deps-conda-dev-env.yml
+     conda activate omnisci-dev
 ```
 
-When pulling changes from upstream, be sure to update submodule references using `git submodule update`.
+### Compilation
 
 If using a Conda enviornment, run the following to build and install HDK:
 
-```
-mkdir build
-cd build
-cmake -DENABLE_CONDA=on ..
-make -j
-make install
+```bash
+    mkdir build && cd build
+    cmake -DENABLE_CUDA=off ..
+    make -j 
+    make install
 ```
 
-### Dependencies 
+### Issues
 
-Conda environments are used for HDK development. Use the yaml file in `scripts/conda`:
-
-```
-     conda env create -f scripts/conda/hdk-conda-dev-env.yml
-     conda activate hdk-dev
-```
+If you meet issues during build refer to [`.github/workflows/build.yml`](https://github.com/intel-ai/hdk/blob/b0bc9ca3f599f648bcc1a8e1eb49c0f02e1f7aca/.github/workflows/build.yml)
 
 ## Test
 
 Python tests can be run from the python source directory using `pytest`. 
+
+```bash
+     pytest python/tests/ 
```

Comment:
You mentioned it fails due to the missing Modin dependency. If we are fixing the documentation, then it would be nice to include all the required steps to successfully run tests.

Reply:
Fixed